packages/cli/docs/custom-plugins.md

```diff
@@ -0,0 +1,637 @@
+# Integrating a plugin in the CLI (based on real examples)
```

Comment:
Question - is the removal of the other file intentional?
https://github.com/code-pushup/cli/pull/312/commits/39b68eefd408e9fd7bd1a16fe4f224d891b9bd4e